CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ and includes an additional section for migration notes.
 - *ORCA-597*
   - Server access logging is now enabled for graphql application load balancer.
 - *ORCA-614*, *ORCA-428* Moved some Internal Reconciliation functionality to GraphQL
+- *ORCA-679* Updated area in recovery where granule ID was treated as a globally unique key. Per Cumulus updates, uniqueness is no granule ID plus collection ID.
```

Comment:
This is the Epic number. Opted to create a side-branch and merge into that to keep develop functional.